index.js

```diff
@@ -33,3 +34,7 @@ function getSlackMentionFromGitHubId (gitHubId) {
   }
   return `<@${id}>`
 }
+
+function getSlackIdFromGitHubId (gitHubId) {
+  return SlackGitHubCacheBuilder.getSlackIdFromGitHubId(gitHubId, slackGitHubCache)
+}
```

Comment:
I think it would be nice if the cache was an object exposing an interface to do conversions. Passing around functions as done with these two can become a bit troubling when functions are renamed or removed. Labels get out of sync. If they were object methods IDEs could detect + warn if they get removed or renamed (I think).

Reply:
Yeah, I think it makes sense to eventually create a service interface for this.

Reply:
Agreed.